challenge_2/c/almyk/src/main.c

```diff
@@ -0,0 +1,66 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#define MAX_SIZE 128
+
+int findSingleInt(int *ary, int n);
+void getInput(int **ary, int *n);
+
+int main(void){
+    // local declarations
+    int *zArr;
+    int result;
+    int n;
+
+// statements
+    getInput(&zArr, &n);
+    result = findSingleInt(zArr, n);
+
+    // if result is less than 0 no single integer was found
+    if(result>=0)
+        printf("Single integer is: %d\n", result);
+    else printf("No single integer found\n");
+
+    return 0;
+} // main
```

Comment:
Proper formatting will make your code read a lot easier.  Make sure you leave empty lines between your methods and your comments match the indentation of it's logical block.